Automattic/wp-calypso
# Comments: add buttons to empty spam and trash

## Proposed Changes

It's currently possible to bulk edit comments in Calypso, but not to empty spam or trash with a single action (like you can in wp-admin). This was spotted by Matt in a Jetpack walkthrough recently - see pedxs5-7f-p2.

This PR adds an "empty" button to the spam and trash tabs in Calypso comment management.

<img width="1003" alt="Screenshot 2023-03-20 at 3 20 09 PM" src="https://user-images.githubusercontent.com/17325/226231926-b2a51d0d-0779-4ea4-812e-fb364f20cec2.png">

## Testing Instructions

1. Ensure there is an 'Empty spam' button on http://calypso.localhost:3000/comments/spam and an 'Empty trash' button on http://calypso.localhost:3000/comments/trash when you have comments in Spam or Trash.
2. Try the button and ensure that it empties all comments from the current section successfully, and gives you adequate information while it does so. The comment counts should be accurate after the emptying has happened.

## Pre-merge Checklist
